pkg/sidecar/sidecar_test.go

```diff
@@ -69,7 +69,7 @@ func TestSidecar_RunDaemon(t *testing.T) {
 		SvidFileName:       "svid.pem",
 		SvidKeyFileName:    "svid_key.pem",
 		SvidBundleFileName: "svid_bundle.pem",
-		Log:                logger.Std,
+		Log:                logrus.New().WithField("system", "spiffe-helper"),
```

Comment:
if it is for testing we must not provide a logger but a fake, to allow testing results